meson.build

```diff
@@ -95,7 +86,6 @@ endif
 if libidn2_dep.found() or libidn_dep.found()
   # Check for libunistring, we need it for psl_str_to_utf8lower()
   libunistring = cc.find_library('unistring')
-  libiconv = cc.find_library('libiconv')
```

Comment:
Is this being removed again on purpose? This was just added in a previous PR #136 

Or was it removed by accident during conflict resolution?

Reply:
The meson.build seems to be auto-generated from time to time, not sure if by 'make' or './configure'. Unnoticed by me this time. Sorry for that, but if that ever happens again, I have to remove `meson.build` from the repo (auto-generated files don't belong there).